src/Controls/tests/TestCases.HostApp/Issues/Issue27200.xaml.cs

```diff
@@ -0,0 +1,93 @@
+﻿
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+namespace Maui.Controls.Sample.Issues;
+
+[Issue(IssueTracker.Github, 27200, "The size of the CollectionView header is incorrect when it contains a Binding on an IsVisible", PlatformAffected.iOS)]
+public partial class Issue27200 : ContentPage
+{
+	Issue27200ViewModel _pageModel;
+	public Issue27200()
+	{
+		InitializeComponent();
+		BindingContext = _pageModel = new Issue27200ViewModel();
+	}
+}
+
+public class Issue27200ViewModel : INotifyPropertyChanged
```

Comment:
When adding screenshots please make this private.

Reply:
Modified